documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md

```diff
@@ -0,0 +1,224 @@
+particular domain. Domain modeling is delegated to shared vocabularies
+and data schemata, as provided e.g. by domain-specific communities of
+the International Data Spaces. The Information Model does not provide a
+meta-model for defining custom datatypes comparable to standards such as
+OData[^2] or OPC-UA[^3]. Concerns beyond the scope of modeling Digital
+Resources and their interchange are considered out of scope. The
+Information Model therefore does not deal with the side effects of data
+exchange (e.g. in scenarios in which data is used for time-critical
+machine operations).
+
+### Model Representations
+
+The Information Model has been specified at three levels of
+formalization. Each level corresponds to a digital representation,
+ranging from this high-level, conceptual document down to the level of
+operational code, as depicted in Figure
+[1](#fig:Representations_of_the_Information_Model){reference-type="ref"
+reference="fig:Representations_of_the_Information_Model"}. Every
+representation depicts the complete Information Model in its particular
+way. Among the different representations, the Declarative Representation
+(IDS Vocabulary) is the only normative specification of the Information
+Model. As such, it is accompanied by a set of auxiliary resources (e.g.
+guidance documents, reference examples, validation tools, and editing
+tools intended to support a competent, appropriate, and consistent usage
+of the IDS Vocabulary).
+
+#### Conceptual Representation
+
+The Conceptual Representation of the Information Model presents a
+high-level overview of the main, largely invariant concepts, with no
+commitment to a particular technology or domain. It targets a general
+audience, management boards, and media, as it provides basic information
+and promotes a shared understanding of the concepts by means of a
+textual document and a plausible visual notation. If available,
+references to related elements of the Declarative Representation and a
+Programmatic Representation are provided, encouraging the reader to take
+a look at these alternative implementations.
+
+#### Declarative Representation
+
+The Declarative Representation (IDS Vocabulary) provides a normative
+view of the Information Model of the International Data Spaces.[^4] It
+has been developed along the analysis, findings, and requirements of the
+Conceptual Representation. Based on a stack of W3C Semantic Web
+technology standards[^5] and standard modeling vocabularies (DCAT[^6],
+ODRL[^7], SKOS[^8], etc.), it provides a formal, machine-interpretable
+specification of concepts envisaged by the Conceptual Representation,
+residing at the persistent namespace URI <https://w3id.org/idsa/core/>.
+Furthermore, it details and formally defines entities of the
+International Data Spaces in order to be able to share, search for, and
+reason upon the structured metadata describing these entities. As such,
+it comprises a complete referential model allowing the derivation of a
```

Comment:
```suggestion
reason upon the structured metadata describing these entities. The IDS Vocabulary is defined using RDF Schema[^rdfs] and the OWL Web Ontology Language[^owl]; additionally, descriptions of Digital Resources can be validated against SHACL shapes[^shacl] that express syntactic and semantic conditions. Queries against, e.g., Data Resources listed in the Data Catalogue of a Connector or Broker, or against Software Resources available from an App Store, can be formulated in query languages such as SPARQL[^sparql].  Thus, the
Declarative Representation comprises a complete referential model allowing the derivation of a
```
Now also explicitly referencing the basic W3C standards mentioned in the figure.